Shared/Classes/Controller/Rights/OTGeolocationRightsViewController.m

```diff
@@ -96,7 +99,7 @@ - (void)setupGoogleAutocompleteFetcher {
     
     // Set up the autocomplete filter.
     GMSAutocompleteFilter *filter = [[GMSAutocompleteFilter alloc] init];
-    filter.type = kGMSPlacesAutocompleteTypeFilterAddress;
+    filter.type = kGMSPlacesAutocompleteTypeFilterRegion;
```

Comment:
Il me semble que, pour Entourage, on veut permettre les adresses plus précises (ex : une station de métro ou une rue). On devrait peut-être essayer `kGMSPlacesAutocompleteTypeFilterGeocode`.

Je crois aussi que pour PFP on veut uniquement des adresse exactes, donc que `kGMSPlacesAutocompleteTypeFilterAddress` est ce qu'on veut.

Tu peux vérifier avec Vaïté ?

Je vois aussi que l'app Android n'applique apparemment aucun filtre :
https://github.com/ReseauEntourage/entourage-android/blob/Version_561847/app/src/main/java/social/entourage/android/location/LocationFragment.java#L397-L400

|||
-|-
kGMSPlacesAutocompleteTypeFilterGeocode | Geocoding results, as opposed to business results.
kGMSPlacesAutocompleteTypeFilterAddress | Geocoding results with a precise address.
kGMSPlacesAutocompleteTypeFilterRegion | Results that match the following types: "locality", "sublocality"  "postal_code", "country", "administrative_area_level_1",  "administrative_area_level_2".